pages.pt-BR/common/ab.md

```diff
@@ -0,0 +1,20 @@
+# ab
+
+> Ferramenta de benchmarking do servidor Apache. A ferramenta mais simples para realizar testes de carga.
+> Página oficial: <https://httpd.apache.org/docs/2.4/programs/ab.html>.
+
+- Executar 100 requisições HTTP do tipo GET para uma determinada URL:
+
+`ab -n {{100}} {{url}}`
+
+- Executar 100 requisições HTTP do tipo GET para uma determinada URL. Sendo que as requisições concorrentes acontecerão em grupos de 10:
+
+`ab -n {{100}} -c {{10}} {{url}}`
+
+- Utilizar a funcionalidade HTTP _Keep Alive_, permitindo que várias requisições sejam feitas em uma sessão HTTP:
```

Comment:
```suggestion
- Utilizar a funcionalidade HTTP Keep Alive, permitindo que várias requisições sejam feitas em uma sessão HTTP:
```

Reply:
@mebeim Sorry, I didn't remember that was more than one.